contracts/Conference.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.4.2;
+
+
+contract Conference {
+
+    string name;
+    string url;
+    string date;
+    string venue;
+    string freezeTime;
```

Comment:
In solidity, date/time can't be taken as input. Thoughts please..

Reply:
It's not that you cannot take date/time as input, you can always take them in, as strings. It's just that the compiler doesn't understand '16 April, 2019' as a date. For our purpose, I think we only need them for display (as of now), and not for any computations. So, for the timing, having them as strings should do the job. Later on, we can always make necessary changes